### PR TITLE
Adding Pre-GSoC Journey Blog Post

### PR DESCRIPTION
**Pull request by ganimtron-10**
This is the first blog post describing the journey before getting selected into GSoC-22.

### CODE REVIEW

File: docs/source/posts/2022/2022-05-25-pre-gsoc-journey-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
... to code basic javascript programs such as addition, subtraction and do manipulations using variables. In my vacation while I was searching for a gameplay video on Youtube, A video titled “How to make a Game” by Brackeys grabbed my attention and made me click on it. I watched the whole video but was unable to understand the most of the things ie. transform, rigidbody, mesh, collider etc. But it generated an internal feeling that I can also create a game. I tried to replicate the process in the video but when I went on to the Official Website of Unity(Game Engine) the whole package size was around 1.5 Gigabytes which was a huge sum at that time(the time when 2G ruled and 100 Megabytes was for 10 INR). So I was unable to work on it. But still some unknown inspiration motivated me to watch the whole “How to make a Game” playlist of 10 videos. As I moved ahead some concepts were getting cleared and I had a surface level idea how to make things work in game(Not how games actually worked). As I was pretty much immersed into game development my feeds started showing “How to create a 3D character”, “How to Animate” videos which led me to discover Blender, 3Ds Max, etc. animation softwares.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-05-25-pre-gsoc-journey-praneeth.rst:23]: remove full-stop(.) after ie

File: docs/source/posts/2022/2022-05-25-pre-gsoc-journey-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
...uted. I was fascinated by viewing the visuals of the Earth with the orbiting moon and the satellite animation (I had doubts whether my laptop will be able to handle these computations or not, but till now It hasn’t proved me wrong in most of the cases). I checked the code then and within 200 lines of code all this was all possible. That made me understand the real power of FURY. After this I checked for some of the good first issues but as it was GSoC period most of the issues were under process so I tried to learn more about the tutorials and the code. There I found a few documentation errors which I fixed as my first PR. I also pointed out some issues which I faced while working with FURY. FURY has a great discord community which engages newbies and solves most of their issues. After two months GSoC Contributors Applications started and I also tried to apply for the same. I created a proposal on Improving the UI system but due to lack of experience and as I was unable to prove my coding skills, I was rejected. But this didn’t disengage me from FURY. I loved the simplicity by which each part was created. Then as time passed I worked on some of the issues and created some more PRs.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-05-25-pre-gsoc-journey-praneeth.rst:38]: `I tried to contact, regarding this issue with the community and also mailed an old GSoC student but they too were unable to solve the query` you can try to rewrite it like this: I tried to contact them regarding this issue and also mailed an old GSoC student but they too were unable to solve the query

**Review by xtanion — COMMENTED** [review]
Nice blog @ganimtron-10 
I found a few typos and added my suggestions below PTAL.
Thanks

**Comment by ganimtron-10** [comment]
Thank You @m-agour and @xtanion for such a helpful review!

**Review by skoudoro — APPROVED** [review]
Hi @ganimtron-10,

Thank you for this first blog post. Overall, it looks good. Some sentences could be improved (strange English sometimes) but that's fine. we will see improvement over weeks with the future blog posts.

merging

**Pull request merged**